client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -76,9 +80,14 @@ const ImageSection = styled.div`
 `;
 
 const Header = styled( FormattedHeader )`
+	margin: 0 0 24px 0 !important;
 	.formatted-header__title {
-		font-size: 2.25rem;
+		font-size: 2.75rem;
 		line-height: 3rem;
+		text-wrap: nowrap;
```

Comment:
The header is doing interesting wrapping because it contains a <br> element. I'd like to remove that <br>, but that will require a "string freeze" and new translations. So, I'll propose that in a follow-up PR.

Reply:
I updated this so the header wraps on mobile. Once the `<br>` element is removed we can remove this or replace with `text-wrap: pretty;`
```
 @media ( min-width: 400px ) {
	text-wrap: nowrap;
}
```